client/landing/stepper/declarative-flow/internals/steps-repository/import/index.tsx

```diff
@@ -17,47 +15,42 @@ import { generateStepPath } from './helper';
 import type { Step } from '../../types';
 import './style.scss';
 
-const isEnabledImportLight = isEnabled( 'onboarding/import-light-url-screen' );
-
 export const ImportWrapper: Step = function ( props ) {
 	const { __ } = useI18n();
 	const { navigation, children, stepName, flow } = props;
 	const currentRoute = useCurrentRoute();
 	const [ , setMigrationConfirmed ] = useMigrationConfirmation();
 	const shouldHideBackBtn = currentRoute === `${ IMPORT_FOCUSED_FLOW }/${ BASE_ROUTE }`;
+
 	const getSkipLabelText = () => {
-		if ( flow === IMPORT_HOSTED_SITE_FLOW ) {
-			return __( 'Create a site' );
-		}
+		switch ( flow ) {
+			case IMPORT_HOSTED_SITE_FLOW:
+				return __( 'Create a site' );
 
-		return flow === IMPORT_FOCUSED_FLOW
-			? __( 'Skip to dashboard' )
-			: __( "I don't have a site address" );
-	};
+			case IMPORT_FOCUSED_FLOW:
+				return __( 'Skip to dashboard' );
 
-	const getGoNext = () => {
-		if ( flow === IMPORT_HOSTED_SITE_FLOW ) {
-			return () => window.location.assign( '/setup/new-hosted-site' );
+			default:
+				return __( "I don't have a site address" );
 		}
-
-		return navigation.goNext;
 	};
 
-	useEffect( () => setMigrationConfirmed( false ), [] );
-
-	const shouldHideSkipBtn = () => {
+	const getGoNext = () => {
 		switch ( flow ) {
-			case IMPORT_FOCUSED_FLOW:
-				return currentRoute !== `${ flow }/${ BASE_ROUTE }`;
-
 			case IMPORT_HOSTED_SITE_FLOW:
-				return currentRoute !== `${ flow }/${ BASE_ROUTE }`;
+				return () => window.location.assign( '/setup/new-hosted-site' );
 
 			default:
-				return currentRoute !== `${ flow }/${ BASE_ROUTE }` || isEnabledImportLight;
+				return navigation.goNext;
 		}
 	};
 
+	const shouldHideSkipBtn = () => {
+		return currentRoute !== `${ flow }/${ BASE_ROUTE }`;
```

Comment:
We probably want to reconsider the condition here. Previously we have `isEnabledImportLight` and this is always true due to the feature flag, so by default it always returns `true`. 
```
return currentRoute !== `${ flow }/${ BASE_ROUTE }` || isEnabledImportLight;
```

But now since we're only checking `currentRoute !== `${ flow }/${ BASE_ROUTE }` we might get some unexpected results, see a demo video below:

https://github.com/Automattic/wp-calypso/assets/4074459/333e3848-41d9-4db7-a939-8708cc86f9f7

We're seeing `I don't have a site address` text in those processing steps, and there were not there before.